docs/source/model_design/vivarium_features/other_causes_ylds/index.rst

```diff
@@ -0,0 +1,230 @@
+
+For a given vivarium simulation, we have typically developed individual cause 
+models for specific conditions. Conditions to include in a simulation are 
+selected based on relevance to the research question and are not exhaustive of 
+all conditions modeled in GBD. This poses a question of how to handle morbidity 
+and mortality due to causes not explicitly modeled within our simulation 
+("background" causes). Historically, vivarium has handled background morbidity 
+differently from background mortality, as described below.
+
+Background mortality
+~~~~~~~~~~~~~~~~~~~~
+
+Vivarium automatically models background mortality in addition to mortality due 
+to modeled causes as mortality due to :code:`other_causes`. The cause-specific 
+mortality rate of :code:`other_causes` is equal to:
+
+.. math::
+
+  CSMR_\text{other causes} = ACMR - \sum_{c=i}^n CSMR_c
+
+Where: :math:`ACMR` is the all-cause mortality rate and :math:`CSMR_c` is the 
+cause-specific mortality rate of a modeled cause :math:`c`.
+
+Background morbidity
+~~~~~~~~~~~~~~~~~~~~
+
+Historically, vivarium has **not** modeled background morbidity. Rather, by 
+default, vivarium models morbidity due to modeled causes only. Among the 
+modeled causes, a partial comorbidity adjustment is performed.
+
+Specifically, for a simulant that possesses multiple causes of disability 
+simultaneously, the YLDs accrued for that simulant in a given timestep are 
+equal to:
+
+.. math::
+
+  YLDs_\text{all causes} = (1 - \prod_{c=i}^n (1 - DW_c)) * \text{time step scalar}
+
+.. math::
+
+  YLDs_\text{cause-specific} = DW_c * \text{time step scalar}
+
+Vivarium can then observe YLDs due to all *modeled* causes AND/OR 
+cause-specific YLDs accrued in the simulation. Notably, the sum across 
+cause-specific YLDs will be greater than the "all-cause" YLDs in vivarium 
+simulation outputs using this methodology.
+
+Problem space
++++++++++++++
+
+There are multiple problems with the default vivarium behavior not to model 
+background morbidity, including:
+
+- Comorbidity adjustment issues
+
+  - Cannot calculate cause-specific YLDs adjusted for comorbidity even just among modeled causes, resulting in overestimation of cause-specific YLDs relative to "all cause" YLDs among modeled causes
+
+  - Does not adjust for comorbidity due to unmodelled causes, resulting in overestimation of YLDs due to modeled causes relative to GBD estimates
+
+- Underestimation of total YLDs
+
+  - Only observe a subset of total YLDs within our simulation. While we can calculate YLDs averted between scenarios, we cannot accurately calculate percent reduction in all-cause YLDs or DALYs relative to baseline because we do not model YLDs due to all causes at baseline.
+
+  - Causes us to overestimate impact of a death averted in our simulation. Death averted some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline. 
```

Comment:
```suggestion
  - Causes us to overestimate impact of a death averted in our simulation. Death averted allow some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline. 
```
Not sure if this is the best word choice but a verb seems to be missing here? Unless I am misreading this sentence.